Dockerfile.downstream

```
@@ -1,27 +1,30 @@
+ARG TARGETARCH
```

```suggestion
ARG TARGETARCH
ARG COMMIT
```

Shouldn't it be on the top ? :thinking:

---

Done! Thanks